src/types/currency.ts

```diff
@@ -85,3 +94,12 @@ export function currencyIdToMonetaryCurrency<U extends CurrencyUnit>(currencyId:
     }
     throw new Error("No CurrencyId entry for provided ticker");
 }
+
+export function currencyIdToLiteral(currencyId: InterbtcPrimitivesCurrencyId): CurrencyIdLiteral {
+    return tickerToCurrencyIdLiteral(currencyIdToMonetaryCurrency(currencyId).ticker);
```

Comment:
Minor, but I find this quite hard to read! Would be a bit easier to follow if `currencyIdToMonetaryCurrency(currencyId).ticker` was assigned to a variable rather than invoked directly in the function.